### PR TITLE
Feature/bbs 14 blocked users

### PR DESCRIPTION
**Pull request by tripox**

### CODE REVIEW

File: modules/aleph/lib/AlephClient/AlephClient.class.php [hunk 1]
```diff
@@ -132,33 +132,86 @@ public function authenticate($bor_id, $verification, $library = 'ICE53', $sub_li
         'pass' => $verification,
       );
 
-      if ($res) {
+      // Check if the user is blocked.
```
Comment (danquah) [inline modules/aleph/lib/AlephClient/AlephClient.class.php]: One extra line of comment here noticing that this is ids for sub libraries would be nice

File: modules/aleph/lib/AlephClient/AlephClient.class.php [hunk 2]
```diff
@@ -132,33 +132,86 @@ public function authenticate($bor_id, $verification, $library = 'ICE53', $sub_li
...
+      $block_codes = array(
+        'z305-delinq-1' => 'z305-delinq-n-1',
+        'z305-delinq-2' => 'z305-delinq-n-2',
+        'z305-delinq-3' => 'z305-delinq-n-3',
+      );
+
+      // Loop through denlinq-1, denlinq-2, denlinq-3.
```
Comment (danquah) [inline modules/aleph/lib/AlephClient/AlephClient.class.php]: Just "sub libraries" otherwise you have to update the comment each time you alter the list

**Review by danquah — CHANGES_REQUESTED** [review]
Almost there :)

**Comment by tripox** [comment]
Fixed, @danquah 😊

**Review by danquah — APPROVED** [review]

**Pull request merged**